packages/shared-ux/chrome/navigation/src/ui/components/navigation_section_ui.tsx

```diff
@@ -147,30 +138,25 @@ export const NavigationSectionUI: FC<Props> = ({ navNode, items = [] }) => {
     : {};
 
   return (
-    <EuiCollapsibleNavGroup
+    <EuiCollapsibleNavItem
       id={id}
       title={title}
-      iconType={icon}
-      iconSize="m"
-      isCollapsible
-      initialIsOpen={isActive}
-      onToggle={(isOpen) => {
-        setIsCollapsed(!isOpen);
-        setDoCollapseFromActiveState(false);
+      icon={icon}
+      iconProps={{ size: 'm' }}
+      accordionProps={{
+        initialIsOpen: isActive,
+        onToggle: (isOpen) => {
+          setIsCollapsed(!isOpen);
+          setDoCollapseFromActiveState(false);
+        },
+        ...navNode.accordionProps,
       }}
       forceState={isCollapsed ? 'closed' : 'open'}
       data-test-subj={`nav-bucket-${id}`}
       {...propsForGroupAsLink}
-    >
-      <EuiText color="default">
-        <EuiSideNav
```

Comment:
Note: `EuiSideNav` was previously doing some opinionated work in the background automatically to expand accordion items that had `isSelected` children. Now that we're not using `EuiSideNav`, we need to move that work to the Kibana side. See https://github.com/elastic/kibana/pull/164910/files#r1334691666